lms/templates/courseware/info.html

```diff
@@ -65,7 +66,14 @@ <h2 class="hd hd-2 page-title">
                 ${_("Welcome to {course_title}!").format(course_title=course_title)}
               % endif
               % if show_subtitle:
-                <div class="page-subtitle">${course_subtitle}</div>
+                <%
+                  display_course_subtitle = course_subtitle
```

Comment:
@taranjeet There's no need to introduce another variable here, I think. You can just remove this line and do

```python
course_subtitle = ccx.display_name
```

below.

Reply:
@itsjeyd : This is fixed. Thanks for pointing it out!

Reply:
@itsjeyd : I think we have to live with this only. From the [docs](http://docs.makotemplates.org/en/latest/runtime.html#context-variables), we can see that this requires changing the way context is passed into render. 

> Another facet of the Context is that its dictionary of variables is immutable

PS: Read the whole part after this.upto heading "Context Methods and Accessors".

If there is any suggestion or I am missing something, please let me know.

Thanks :)

Reply:
@taranjeet Ah, OK! That's a subtlety I wasn't aware of. Thanks for clarifying, there's no need to change this, then.